### ISSUE TITLE
The comparison of OVSeg model shown in Figure 5 in the paper, what training set and validation set is used for OVSeg model here? Convenient to inform! thank you

### ISSUE THREAD

**Issue by QY1994-0919** [dvlab-research/LISA#183]
The comparison of OVSeg model shown in Figure 5 in the paper, what training set and validation set is used for OVSeg model here? Convenient to inform! thank you
